_faq/cep-and-its-importance.md

```diff
@@ -0,0 +1,12 @@
+---
+question: "What is a cEP and why it is so important to do analysis pre-gsoc and design as early as possible, to be finalized in the cEP?"
+---
+A **cEP (coala Enhancement Proposal)** is proposed to the coala community discussing the new major features that the contributor intends to implement during GSoC.
+
+**cEP's** are designed before implementation. Implementing major features usually happens from a technical perspective and may lead to poorly designed content - developer and user wise. This can essentially save a lot of coding work when big features are written which are oftentimes not well enough designed initially. This also makes sure that major features are properly documented.
```

Comment:
`when big features are written which are oftentimes not well enough designed initially` is already conveyed effectively through the sentence before, so you can probably cut this for brevity. Suggested change: `This can essentially save a lot of coding work and also makes sure that major features are properly documented.`